### DISCUSSION TITLE
Anchor to another markdown file

### DISCUSSION THREAD

**Discussion by Marinorama** [gsantner/markor#1533] [Q&A] [answered]
I tried to use anchors linking to headers and it works within a markdown file. Linking to a header in another file seems impossible. Tried various pathnames written before the # anchor link. Is there a specific method or is it really not working in Markor?

**Comment by gsantner** [comment, marked as answer]
It is not implemented, if you want to help (programming) feel free to do if you know how to, or want to learn it.

<https://github.com/gsantner/markor/issues/912>
